django/utils/functional.py

```diff
@@ -311,6 +308,16 @@ def __deepcopy__(self, memo):
     __contains__ = new_method_proxy(operator.contains)
 
 
+def unpickle_lazyobject(wrapped):
+    """
+    Used to unpickle lazy objects.
+
+    Actually just returns its argument, which will be the wrapped object.
+    """
+    return wrapped
+unpickle_lazyobject.__safe_for_unpickling__ = True
```

Comment:
Note that this line is unnecessary. Checks for `__safe_for_unpickling__` were removed in Python 2.3: https://docs.python.org/2/whatsnew/2.3.html?highlight=__safe_for_unpickling__#pep-307-pickle-enhancements